tools/homebrew-start/lanraragi

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+trap "kill 0" SIGINT
+export LANDATADIR="<LANDATADIR>"
```

Comment:
`LRR_DATA_DIRECTORY` would be a better name for the env variable -- It stays coherent with the already existing `LRR_NETWORK` variable.

Reply:
Yep, I noticed that too :) 
But I changed it before reading your comment. It is now called `LRR_DATADIR`. Do you want `LRR_DATA_DIRECTORY`?